{"version":1,"defects":{"Error":4,"Tests\\Unit\\RouterTest::test_route_is_registered":1,"Tests\\Unit\\RouterTest::it_registers_routes with data set #0":4,"Tests\\Unit\\RouterTest::it_registers_routes with data set #1":4,"Tests\\Unit\\RouterTest::it_registers_routes with data set #2":4,"Tests\\Unit\\RouterTest::it_registers_routes with data set #3":4,"Tests\\Unit\\RouterTest::it_registers_routes with data set #4":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #0":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #1":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #2":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #3":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #4":4,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #5":4,"Tests\\Unit\\RouterTest::it_throws_route_not_found":4,"Tests\\Unit\\RouterTest::it_throws_invalid_route_exception":3},"times":{"Error":0,"Tests\\Unit\\RouterTest::test_route_is_registered":0.001,"Tests\\Unit\\RouterTest::it_registers_routes with data set #0":0.003,"Tests\\Unit\\RouterTest::it_registers_routes with data set #1":0,"Tests\\Unit\\RouterTest::it_registers_routes with data set #2":0,"Tests\\Unit\\RouterTest::it_registers_routes with data set #3":0,"Tests\\Unit\\RouterTest::it_registers_routes with data set #4":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #0":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #1":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #2":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #3":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #4":0,"Tests\\Unit\\RouterTest::it_registers_routes with data set #5":0,"Tests\\Unit\\RouterTest::it_resolves_routes with data set #5":0,"Tests\\Unit\\RouterTest::it_throws_route_not_found":0.001,"Tests\\Unit\\RouterTest::it_throws_route_not_found_exception":0.001,"Tests\\Unit\\RouterTest::it_throws_invalid_route_exception":0}}